src/components/cards/TripCard/TripCard.styles.js

```diff
@@ -0,0 +1,14 @@
+export const root =
+  'su-group su-relative su-overflow-hidden su-bg-black-true su-break-words su-border-su-border-black';
```

Comment:
```suggestion
  'su-group su-relative su-overflow-hidden su-bg-black-true su-break-words su-border-black';
```
Looks like a typo perhaps?